doc/setup/installing_linux_python.adoc

```diff
@@ -26,27 +45,22 @@ git clone https://github.com/OpenSimulationInterface/open-simulation-interface.g
 cd open-simulation-interface
 ----
 +
-. Create a new virtual environment.
-+
-----
-virtualenv -p python3 venv
-----
-+
-. Activate the virtual environment.
+. Optionally create and activate a new virtual environment.
 +
 ----
+python3 -m venv venv
 source venv/bin/activate
 ----
 +
 . Install Open Simulation Interface.
 .. Local installation
 +
 ----
-python3 -m pip install .
```

Comment:
I thought you only use it with the prefix: python3 -m pip*

Reply:
Only in specific circumstances (e.g. upgrading pip in place, or where there was a multitude of python installations 2 vs 3 - also somewhat ubuntu specific); on most systems pip install is the suggested way to do stuff, and generally is e.g. the way PyPI recommends it.

Both ways work, but as there already was a mixture between the two, I picked one to stay consistent.